advanced/is-pattern-2/templates/analytics-worker/identity-server-analytics-worker-1-statefulset.yaml

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@darsham3 why have we defined 2 StatefulSet resources for the Analytics Worker deployment?

We can ideally handle this via a single definition. If the configs differ in each instance, let's try to cover them via environment variables. Please see the [sample](https://github.com/wso2/kubernetes-apim/tree/v3.2.0.2/advanced/am-pattern-1/templates/am-analytics/worker) in APIM Analytics.

Reply:
updated